utils/supabase/server.ts

```
@@ -22,4 +61,76 @@
+}
+
+/**
+ * Creates a Supabase client for use in Next.js middleware, handling cookies via request and response headers.
+ *
+ * Parses cookies from the incoming request and provides a response object with any new cookies set in the `Set-Cookie` header.
+ *
+ * @param request - The incoming HTTP request object containing cookies in its headers.
+ * @returns An object containing the Supabase client and a response with updated cookie headers.
+ */
+export function getSupabaseMiddleware(request: Request) {
+  let response = new Response();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'response' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)